### PR TITLE
add support for go-like log lines

### PR DESCRIPTION
**Pull request by a8m**
@eyal-stratoscale if you want to test it, clone, checkout, and update your extension

### CODE REVIEW

File: src/interpreter.js [hunk 1]
```diff
@@ -196,9 +206,24 @@ function jsonLineToText(json) {
...
-  LOG_LEVELS[obj.levelname].count += 1
-
-  return [ [ created(obj.created), threadName(obj.threadName), levelname(obj.levelname, MAX_LEVEL_WIDTH), " ", msg, exc_text, fileLocation(obj.pathname, obj.lineno) ].join("") , obj.levelname ]
+  // if it's a Python-like log line
+  if (obj.threadName && obj.pathname) {
+    LOG_LEVELS[obj.levelname ].count++;
+    return [ [ created(obj.created), threadName(obj.threadName), levelname(obj.levelname, MAX_LEVEL_WIDTH), " ", msg, exc_text, fileLocation(obj.pathname, obj.lineno) ].join("") , obj.levelname ]
+  }
+  // otherwise, it could be Go application. (e.g: maestro project)
+  LOG_LEVELS[obj.level.toUpperCase()].count++;
```
Comment (eyal-stratoscale) [inline src/interpreter.js]: I think this could be a common line for both if statement

**Review by eyal-stratoscale — COMMENTED** [review]
Nice!
On comment..
Another question, did you have a look if it is possible to make this more efficient?
Thanks!

File: src/interpreter.js [hunk 2]
```diff
@@ -3,12 +3,16 @@ var LOG_LEVELS = {
   "DEBUG"    : { "show": true, "count": 0 },
   "INFO"     : { "show": true, "count": 0 },
   "WARNING"  : { "show": true, "count": 0 },
+  "WARN"     : { "show": true, "cont":  0 },
```
Comment (motty-stratoscale) [inline src/interpreter.js]: cont

**Comment by eyal-stratoscale** [comment]
@a8m works like a charm

**Review by eyal-stratoscale — APPROVED** [review]

**Pull request merged**